README.md

```diff
@@ -9,7 +9,7 @@
 [![Stack Overflow][stack-shield]](https://stackoverflow.com/tags/kontent-ai)
 [![Discord][discord-shield]](https://discord.gg/SKCxwPtevJ)
 
-This package provides you with tools to transform rich text element value from Kontent.ai into a JSON tree and optionally to [portable text standard](https://github.com/portabletext/portabletext).
+This package provides utilities for transforming Kontent.ai rich text into structured formats suitable for resolution and rendering in various environments.
```

Comment:
I am missing some high level overview diagram.
Does something like this make sense?
<img width="1134" alt="image" src="https://github.com/user-attachments/assets/51da3bfa-bf5d-4545-abd8-14a208d40fb8" />

Reply:
I would swap `Mapi` for `HTML` in the external libraries section. Also, `nodesToPortableText` should point to `Array<PortableTextObject>`. `transformNodes` (`transformNodesAsync`) is the one resulting in custom output. fair point adding a visual overview to clarify the logic